### ISSUE TITLE
Update RNGH, Screens, and Reanimated

### ISSUE THREAD

**Issue by brentvatne** [expo/expo#2359]

**Comment by tsapeta** [comment]
Hey @brentvatne, I'm wondering if we could make git submodules for these projects? It might be easier to update them in the future 🤔

**Comment by brentvatne** [comment]
heya! it's pretty easy to update them right now, we just run some scripts for it, eg: https://github.com/expo/expo/blob/a0f4f6bcd3d265a3731ca0cfb2eb1305305d39f1/tools/gulpfile.js#L159-L189

**Comment by brentvatne** [comment]
I'll document the current process - I have found it to be really painless but if you prefer submodules that's fine! I usually try to avoid submodules when I can as I don't particularly enjoy that workflow, but you work on the sdk more so feel free to make changes that suit you best.

Good point about not knowing which commit we pulled from. Usually it's just master on the expo fork, but that'll only be useful for the most recent version. We could add tags also if we stick with this approach.
